src/Core/App.php

```diff
@@ -74,6 +74,92 @@ public static function className($class, $type = '', $suffix = '')
     }
 
     /**
+     * Returns the plugin split name of a class
+     *
+     * Examples:
+     *
+     * ```
+     * App::pluginSplitName(
```

Comment:
This method name doesn't sit well with me. This method is converts from a classname to a short class. Do any of the following feel better:

* shortClass()
* shortName()
* pluginName()